src/org/gk/slicing/SlicingEngine.java

```diff
@@ -1269,12 +1517,14 @@ public static void main(String[] args) {
             String pwd = properties.getProperty("dbPwd");
             if (pwd == null || pwd.trim().length() == 0)
                 pwd = getInput("Please input the password");
+            
+            // Target database.
             String targetDbHost = properties.getProperty("slicingDbHost");
             if (targetDbHost == null || targetDbHost.trim().length() == 0)
                 targetDbHost = getInput("Please input the slice databse host");
-            String targetdbName = properties.getProperty("slicingDbName");
-            if (targetdbName == null || targetdbName.trim().length() == 0)
-                targetdbName = getInput("Please input the slice database name");
+            String targetDbName = properties.getProperty("slicingDbName");
```

Comment:
This is OK since it is inside so called camel naming

Reply:
Understood.